### PR TITLE
Update ringphp to 1.1.0 for PHP 7 compatibility

### PR DESCRIPTION
**Pull request by LukasReschke**
Required as otherwise this will lead to a fatal PHP error.

@DeepDiver1975 THX - will fix:

```
10:40:08 .............................................PHP Fatal error:  Redefinition of parameter $_ in /var/jenkins/workspace/server-master-linux-php7/database/sqlite/label/SLAVE/3rdparty/guzzlehttp/ringphp/src/Client/StreamHandler.php on line 313
```

### CODE REVIEW

**Comment by LukasReschke** [comment]
Core PR is at https://github.com/owncloud/core/pull/17725

**Comment by LukasReschke** [comment]
This avoids updating GuzzleHttp from 5 to 6 which is a BC break and a pain, fixes https://github.com/owncloud/core/issues/16642

File: composer/installed.json [hunk 1]
```diff
@@ -2518,5 +2418,105 @@
             "sftp",
             "storage"
         ]
+    },
+    {
+        "name": "nikic/php-parser",
+        "version": "v1.3.0",
```
Comment (DeepDiver1975) [inline composer/installed.json]: Hmmm ... phpparser changed from 1.1.0 to 1.3.0 - but no code changes? :fish:y

Comment (LukasReschke) [inline composer/installed.json]: It's quite easy. The last PR where we bumped it forget that so the lock file (or so) has already 1.3.0 and 1.3.0 is installed but installed.json lied at us :speak_no_evil:

Comment (DeepDiver1975) [inline composer/installed.json]: :facepalm:

Comment (LukasReschke) [inline composer/installed.json]: Yeah. Me is a big hero :speak_no_evil:

**Pull request merged**